lms/envs/common.py

```diff
@@ -1809,3 +1818,74 @@
     "HarvardX/SW12.9x/3T2014",
     "HarvardX/SW12.8x/3T2014",
 ])
+
```

Comment:
I made a similar comment in cms/envs/common.py above -- it would be nice if there was a way to 'package' these configuration parameters more neatly -- say in a notifications_config.py module, and then import into common.py.  Not a requirement in the case of this PR, but thought I would propose it.